source/sync/data-model/update-schema.txt

```diff
@@ -376,85 +281,36 @@ to both the server-side schema and the client-side object model.
-   - :ref:`Swift: Remap a Property <swift-rename-property>`
-
-A breaking server-side schema change requires you to :ref:`terminate sync 
-<terminating-realm-sync>` in the backend, and then :ref:`re-enable 
-<enable-sync>` sync. Breaking schema changes prevent applications from 
-automatically recovering from a :ref:`client reset <handle-a-client-reset>`.
-As a result, we do not recommend making breaking server-side schema changes.
-
-Instead, you can let the existing collection remain in place with the 
-existing schema, and create a partner collection with the new schema. 
-
-A partner collection is a collection that contains the same data as
-the original collection, but has the new schema definition in place. Partner
-collections use database triggers to ensure that data flows in both directions,
-meaning that when one collection is written to, the other is also written to 
-(with the data modifications required for the new schema).
-
-To implement a breaking schema change using the partner collection strategy, 
-see :ref:`<migrate-schema-partner-collection>`.
+Instead of changing an property's name, consider implementing the 
+:ref:`partner collection strategy <schema-partner_collection-strategy>`, in which 
+you keep the existing collection and schema, and create a new collection 
+with the new schema. 
 
-If you choose to terminate and re-enable sync instead of using the partner
-collection strategy to resolve this type of breaking change, you must 
-manually update the schema. :ref:`Development Mode <development-mode>` 
-does not automatically update your schema for breaking changes.
-
-Handle Existing Atlas Documents
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you choose to change the property's name instead of using the partner collection 
+strategy, you will need to terminate sync, manually update the schema, 
+and re-enable sync. In addition, your client 
+application must perform a :ref:`manual client reset <manual-client-reset>`.
 
-If you choose to terminate and re-enable sync, you must update existing Atlas 
+When you terminate and re-enable sync, you must also update existing Atlas 
 documents to enable them to Sync with your client applications. 
 Without this additional handling, those documents do not Sync and it may 
-appear in the client that the data has been lost.
-
-If an Atlas document maintains the old property name, the entire document no 
-longer syncs to the client application. You could resolve this issue in a 
-few ways:
+appear in the client that the data has been lost. You could resolve this issue 
+in two ways:
 
 - Change the old field name on each document to match the new schema
 - Add a new field to each document with a name that matches the new schema, 
   and copy the value from the old field into it
 
-After you've made these changes, the appropriate documents should once again 
+After you've made these changes, the appropriate documents will 
 sync to the client application.
```

Comment:
Same note as above - avoid tense changes and consider explaining why they sync again?

```suggestion
After you update the documents, they match the client schema and can sync again. 
```